src/Microsoft.AspNetCore.Mvc/project.json

```diff
@@ -16,6 +16,7 @@
     "Microsoft.AspNetCore.Mvc.Formatters.Json": "1.0.0-*",
     "Microsoft.AspNetCore.Mvc.Localization": "1.0.0-*",
     "Microsoft.AspNetCore.Mvc.Razor": "1.0.0-*",
+    "Microsoft.AspNetCore.Mvc.TagHelpers": "1.0.0-*",
```

Comment:
Before now though, Tag Helpers didn't need their own services in DI. This is going to be a pain to use if it's not part of the MVC package, as you'll have to add the Tag Helpers package, and explicitly add the Tag Helper services to DI, and register them in the views. So many steps makes me :cry: so if there's no other objection to this, I say we keep it, or find a way to make the experience of adding and use the Tag Helpers not suck.